andela/ah-olympians-frontend
# chore(setup-react-router-dom) Add router-dom

**What does this PR do?**

- Setting up react-router-dom

**Description of Tasks to be completed**

- Setup react-router-dom
- Testing the functionality with a sample login page

**How should this be manually tested?**

- Clone this repository then navigate to `ft-setup-react-router-164047019`
- Run `npm install` to install dependencies
- Run `npm start` to run the application.
- Change url to `....\login`.It should change without any errors
